eventbridge/platform/aws/publish.js

```diff
@@ -0,0 +1,23 @@
+const { EventBridgeClient, PutEventsCommand } = require("@aws-sdk/client-eventbridge");
+
+const putEvent = async (eventBridgeName, event) => {
+  console.log(`Publishing event to EventBridge: ${eventBridgeName} - ${JSON.stringify(event)}`);
+  const client = new EventBridgeClient({ region: process.env.AWS_REGION });
```

Comment:
Probably better to hoist outside of this method so a single client is created for each process.